Improve formatting and shorten install details

MatthewScholefield opened the pull request:
Python packages can be installed directly via URL.

kanishka-linux commented:
1. About installation, I think it will be good to keep original method along with your suggestion. 
I've written in a similar way in the README of my other project [vinanti](https://github.com/kanishka-linux/vinanti/blob/master/README.md#dependencies-and-installation), which you can take a look at.

2. About using `sudo`, just write some warning or cautionary note as written in the README of vinanti project.

Rest is fine. Thanks for taking interest in the project.

MatthewScholefield commented:
I've just updated to include both versions and a sudo warning. Also extracted the tagline into italics at the top. Let me know if there's anything else you'd like to adjust (or feel free to merge and adjust yourself).
